docs/user_manual.md

```diff
@@ -133,6 +133,8 @@ example that uses all the available configuration keys with an explanation of ea
 
 ```json
 {
+ "addedCapabilities" : [ "IPC_LOCK", "SYSLOG" ],
+ "droppedCapabilities" : [ "SYS_BOOT", "KILL" ],
```

Comment:
sure :)

On Wed, Mar 30, 2016 at 4:30 PM, Matt Brown <notifications@github.com>
wrote:

> In docs/user_manual.md
> <https://github.com/spotify/helios/pull/892#discussion_r57958285>:
>
> > @@ -133,6 +133,8 @@ example that uses all the available configuration keys with an explanation of ea
> >
> >  ```json
> >  {
> > + "addedCapabilities" : [ "IPC_LOCK", "SYSLOG" ],
> > + "droppedCapabilities" : [ "SYS_BOOT", "KILL" ],
>
> forgive the nitpick, but how about "addCapabilities" / "dropCapabilities"
> in the job json so it's not using past tense?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/spotify/helios/pull/892/files/e019aa14389f6e4669b79ba0a921969a4aaa993f#r57958285>
>



-- 
David Xia
Software Engineer
dxia@spotify.com